### DISCUSSION TITLE
Noob here - fr translation

### DISCUSSION THREAD

**Discussion by cbo04** [mozilla/hubs#3752] [General Questions & Discussion] [answered]
Hi there, 

I've translated into French
The fr.json and edited locales file are standing in Pull requests

As a noob in GitHub, I'm not sure I followed the instructions correctly
Could you check please?

Thank you!

**Comment by emclaren** [comment, marked as answer]
Thanks @cbo04! It's amazing to get new translations to Hubs from community members. I'm going to link these pull requests here so they are grouped with this question and then I'll pass it on to my team to have them take a look

https://github.com/mozilla/hubs/pull/3754
https://github.com/mozilla/hubs/pull/3753

Thanks for your work on this
